README.md

```diff
@@ -100,7 +100,7 @@ Examples can be found in [optuna/optuna-examples](https://github.com/optuna/optu
 
 * [AllenNLP](https://github.com/optuna/optuna-examples/tree/main/allennlp)
 * [Catalyst](https://github.com/optuna/optuna-examples/tree/main/pytorch/catalyst_simple.py)
-* [Catboost](https://github.com/optuna/optuna-examples/tree/main/catboost/catboost_simple.py)
+* [Catboost](https://github.com/optuna/optuna-examples/blob/main/catboost/catboost_pruning.py)
```

Comment:
[memo]
The tree based link, https://github.com/optuna/optuna-examples/tree/main/catboost/catboost_pruning.py, is redirected to 
blob based link, https://github.com/optuna/optuna-examples/blob/main/catboost/catboost_pruning.py